src/api/pkg/swagger/models/chart_file.go

```diff
@@ -0,0 +1,173 @@
+package models
```

Comment:
thanks!, that's cruft (this is swagger not deleting old generated stuff if you delete it from the spec)